.github/workflows/docs.yml

```diff
@@ -2,10 +2,6 @@ name: Documentation
 
 on:
   push:
-    branches:
-      - main
```

Comment:
You probably still want to build docs on `main` so that the `dev` docs get built